tests/pending/pos/i10161.scala

```diff
@@ -0,0 +1,18 @@
+object Incompat2 {
```

Comment:
is this meant to be in this PR?

Reply:
Yes, it's an open issue, so we can keep it in the test set